DiceKitTests/Die_Tests.swift

```diff
@@ -404,4 +404,38 @@ extension Die_Tests {
         expect(result.value) == stubbedRoll
     }
     
+    func test_probabilityMass_shouldReturnCorrect() {
+        property("probability mass") <- forAll {
+            (sides: Int) in
+            
+            let inverseSides = -sides
+            
+            let die = Die(sides: sides)
+            let inverseDie = Die(sides: inverseSides)
+            
+            let probMass = die.probabilityMass
+            let inverseProbMass = inverseDie.probabilityMass
+            
+            guard sides != 0 else {
```

Comment:
It needs to be this far down, because I return a check based on `probMass`. Guards don't have to be at the top, they just formalize early return. You can return early whenever, and more than ever they stand out now.